easyrest/views/order_controller.py

```diff
@@ -345,75 +364,88 @@ def change_status(request):
 
     try:
         json = request.json_body
-    except ValueError as e:
+    except ValueError:
         raise HTTPBadRequest("Not valid json")
```

Comment:
The module validator.py already has a JSON format check method. You can use it in the future if you want.